### PR TITLE
Remove doctrine managers

### PR DESCRIPTION
**Pull request by EmmanuelVella**
It seems there is no reason to keep these managers ?

### CODE REVIEW

**Comment by core23** [comment]
Please rebase @EmmanuelVella

**Comment by EmmanuelVella** [comment]
@core23 Rebase done (but I can't test as I'm not using this bundles anymore).

**Comment by core23** [comment]
Thanks @EmmanuelVella 
 
Maybe we can leave both files and add a deprecation notice instead?

**Comment by EmmanuelVella** [comment]
@core23 That a good idea. I have updated my PR.

File: Admin/Manager/DoctrineMongoDBManager.php [hunk 1]
```diff
@@ -19,6 +19,8 @@
 /**
  * this method overwrite the default AdminModelManager to call
  * the custom methods from the dedicated media manager.
+ *
+ * @deprecated
```
Comment (core23) [inline Admin/Manager/DoctrineMongoDBManager.php]: Can you please leave a comment here?

``@deprecated since XXX (to be removed in 3.0)``

**Comment by EmmanuelVella** [comment]
Done !

**Comment by core23** [comment]
Users need to know since when they are deprecated and when they will be removed. @EmmanuelVella

**Comment by core23** [comment]
ping @EmmanuelVella

**Comment by EmmanuelVella** [comment]
I have updated the deprecated message.

File: Admin/Manager/DoctrineMongoDBManager.php [hunk 2]
```diff
@@ -19,6 +19,8 @@
 /**
  * this method overwrite the default AdminModelManager to call
  * the custom methods from the dedicated media manager.
+ *
+ * @deprecated This class is deprecated since version 2.4 and will be removed in 3.0.
```
Comment (greg0ire) [inline Admin/Manager/DoctrineMongoDBManager.php]: 2.4 does not exist.

**Comment by Soullivaneuh** [comment]
@EmmanuelVella According to https://github.com/sonata-project/SonataAdminBundle/issues/3731, sonata media bundle won't be 2.4 but 3.0.

After that, if those manager are really not used, I think we could accept this BC break for 3.0.

I'll ping you when the 3.x will be ready so you will be able to rework this PR. :+1:

**Comment by EmmanuelVella** [comment]
Ho ok I didn't know there won't be a 2.4 version. Waiting for your ping then !

**Comment by Soullivaneuh** [comment]
Thank you @EmmanuelVella, I cherry-picked your commit with deletion instead of deprecation.

https://github.com/sonata-project/SonataMediaBundle/commit/49e628ec497179130988c8aefe83f45d891852a4

**Comment by EmmanuelVella** [comment]
ok :+1:

**Pull request closed without merging**